csrankings-m.csv

```diff
@@ -2500,6 +2500,7 @@ Monika Heiner,Brandenburg University of Technology,https://www-dssz.informatik.t
 Monika Henzinger,IST Austria,https://ist.ac.at/de/forschung/henzinger_monika-gruppe/,NXbggxYAAAAJ
 Monika Rauch,IST Austria,https://ist.ac.at/de/forschung/henzinger_monika-gruppe/,NXbggxYAAAAJ
 Monika Rauch Henzinger,IST Austria,https://ist.ac.at/de/forschung/henzinger_monika-gruppe/,NXbggxYAAAAJ
+Monique Meuschke,University of Magdeburg,https://wasd.urz.uni-magdeburg.de/meuschke/,iMiFnq4AAAAJ

```

Comment:
Home page indicates postdoc, thus ineligible for inclusion

Reply:
Monique's status is *Privatdorzentin* (abbreviation PD), i.e., she got her Habilitation. Thus in the German system she meets all criteria including solely advising PhD students.

Reply:
Needs to be a permanent appointment; please confirm.

Reply:
Yes, it‘s a permanent appointment.